src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_LENGTH = 50;
+    private static final int DEFAULT_CIRCLE_RADIUS = 10;
+    private static final int FIGURE_NUMBER = 5;
+    private static final String DEFAULT_COLOR = Color.WHITE.name().toLowerCase();
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Square getRandomSquare() {

```

Comment:
You should have only 2 public methods in this class - getDefaultFigure and getRandomFigure